source/reference/api/start.txt

```diff
@@ -151,12 +215,101 @@ Request
 
 Behavior
 --------
 
+State
+~~~~~
+
 If the ``start`` request is successful, ``mongosync`` enters the
 ``RUNNING`` state.
+
+Shard Collections
+~~~~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster requires the 
+``sharding`` option, to specify how ``mongosync`` should shard collections.
+
+The ``sharding.shardEntries`` array specifies the collections to shard.
+Collections that are not listed in this array replicate as unsharded.
+
+Supporting Indexes
+~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster can automatically
+create a supporting index for the new shard key on the destination cluster.
+This is done by setting the ``sharding.createSupportingIndexes`` option.
+
+When ``sharding.createSupportingIndexes`` is ``false`` (the default): 
+
+* Each shard key the user provides for the ``sharding.shardEntries`` option
+  must have an existing index on the source cluster.
+
+* One of the indexes used for the shard key must have simple collation if the 
+  collection uses any other collation.
+
+* To use a unique index in the shard key, you must specify its uniqueness when
+  you create the index on the source cluster. 
+
+* Unique indexes on the source cluster that are incompatible with the requested
+  shard key on the destination cluster, such as a unique index on the source
+  that does not contain the requested shard key as a prefix on the destination,
+  can cause ``mongosync`` to fail.
+
+When ``sharding.createSupportingIndexes`` is ``true``: 
+
+* ``mongosync`` synchronizes indexes on the source cluster to the destination.
+
+* If the supporting indexes exist on the source cluster, ``mongosync`` 
+  synchronizes the indexes to the destination cluster and uses them 
+  as the shard key.
+
+* If the supporting indexes don't exist, ``mongosync`` creates them on the
+  destination cluster.
+
+``mongosync`` either creates supporting indexes for all collections or none.
+
+Rename during Sync
+~~~~~~~~~~~~~~~~~~
+
+Collections synchronized from a replica set to a sharded cluster listed in
+the ``sharding.shardEntries`` array, become sharded collections on the
+destination cluster.
+
+Renaming a collection, such as with the :dbcommand:`renameCollection` command
+on the source cluster after calling ``start`` but before ``mongosync`` begins 
+to copy the collection can block the collection from sharding on the destination.
+
+To check whether a collection is safe to rename, call the :ref:`c2c-api-progress`
+endpoint and check the value of the ``collectionCopy.estimatedCopiedBytes`` field 
+in the return document.  
+
+* A value of 0 indicates that ``mongosync`` has not yet begun to copy the
+  collection.  Renaming the collection at this point results in the collection 
+  being unsharded on the destination cluster.
+
+* A value greater than 0 indicates that ``mongosync`` has begun the copy and
+  read the shard map.  Renaming the collection from this point on does not block
```

Comment:
I'm not sure what "read the shard map" means.

Reply:
Fixed